browse-next/src/views/DeviceDiagnosticsSubView.vue

```diff
@@ -454,7 +767,123 @@ const deviceLocationPoints = computed<NamedPoint[]>(() => {
       </div>
     </div>
     <div class="mt-4" v-if="device.type === 'thermal'">
+      <h6 class="mt-4">Device Settings:</h6>
```

Comment:
This really needs a different view for mobile responsive breakpoints – however, I'm also inclined to want to move this all to the Setup tab, since it seems more appropriate there, so I can take care of that if you want.